include/hipSYCL/sycl/queue.hpp

```diff
@@ -741,6 +761,71 @@ class queue : public detail::property_carrying_object
     });
   }
 
+  
+  // Explicit copy functions
+  template <typename T, int dim, access::mode mode, access::target tgt,
+            accessor_variant variant>
+  event copy(accessor<T, dim, mode, tgt, variant> src,
+             shared_ptr_class<T> dest) {
+    return this->submit([&](sycl::handler &cgh) {
+      cgh.copy(src, dest);
```

Comment:
The spec is not really clear how those should behave, but my thinking is that this only makes sense with placeholder accessors, since the user doesn't have access to the command group handler to create non-placeholders.

In this case, you still need to call `require` on the accessor, otherwise it will not be connected to the command group and none of these functions will work :)

Reply:
I've changed all accessors to `access::placeholder` and added the missing `require()` calls.

Reply:
I think you can stick to using `accessor_variant`. In the current implementation it's equivalent to `access::placeholder` but also enables hipSYCL's accessor variant extension :)

Reply:
Done.